Runtime/Scripts/Types/RGRemoteBotList.cs

```diff
@@ -8,4 +8,3 @@ public class RGBotList
         public RGBot[] bots;
```

Comment:
It looks like the types got fixed back, but the files still show as being renamed

Reply:
Oop, yeah, good catch. Updating now.